CMakeLists.txt

```diff
@@ -14,7 +14,9 @@ set(Ginkgo_VERSION_TAG "develop")
 set(PROJECT_VERSION_TAG ${Ginkgo_VERSION_TAG})
 
 # Determine which modules can be compiled
-include(cmake/hip_path.cmake)
+if(NOT DEFINED GINKGO_BUILD_HIP OR GINKGO_BUILD_HIP)
+    include(cmake/hip_path.cmake)
+endif()
```

Comment:
is there any harm to including this every time?

Reply:
no, it avoids introducing many unrelated hip into cmake

Reply:
I guess it's okay for now, we should probably move to native CMake support for HIP in the near future, anyways.